lib/formatters/default.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const path = require('path');
+
+class DefaultPrinter {
+  constructor(options = {}) {
+    this.delegates = [];
+    this.console = options.console || console;
+
+    if (options.json) {
+      options.format = 'json';
+    }
+
+    switch (options.format) {
```

Comment:
I would like this option to support multiple formatters at once. Specifically, something like:

```
ember-template-lint . --format pretty --format json --format xunit
```

And have that run all three types. The majority of the code here actually would support that (since we use `this.delegates` and loop over them), but I think this outer switch statement would need to be tweaked slightly (based on `Array.isArray(options.format)`).

Reply:
Let's do this separately. I can easily add this after this lands.